app/templates/homepage.html

```diff
@@ -15,6 +15,7 @@
           <nav class="section--hero__nav">
             <a href="#about">About</a>
             <a href="#login">Login</a>
+            <a href="#contact">Contact</a>
```

Comment:
Do we really need a separate contact page? It should be something we can add within the home page